packages/core/chrome/core-chrome-browser-internal/src/ui/header/header.tsx

```diff
@@ -105,11 +105,9 @@ export function Header({
   const className = classnames('hide-for-sharing', 'headerGlobalNav');
 
   const Breadcrumbs = <HeaderBreadcrumbs breadcrumbs$={observables.breadcrumbs$} />;
-
```

Comment:
[nit] I think removing this newline lessens readability. Do you mind taking a quick pass of the [final diff](https://github.com/elastic/kibana/pull/150461/files) and restoring any newlines that weren't removed intentionally? For example, the newlines removed in `collapsible_nav.tsx` aren't really meaningful changes and add extra noise/file diffs.

Reply:
I follow [0cb69ae](https://github.com/elastic/kibana/pull/150461/commits/0cb69ae4529e80663c413427e74ddcd2f7d9e1fa) for changes 👍🏻